lib/main.js

```diff
@@ -126,13 +127,16 @@ module.exports = class Router extends EventEmitter {
 
         this.sink = sink ? sink : new SinkMem();
         this.metaStorage = new MetaStorage(this.sink);
+        this.metrics = new Metrics();
 
         this.bundler = new OptimisticBundler({
             env: this.options.env,
             sink: this.sink,
             logger: this.options.logger,
         });
 
+        this.bundler.metrics.pipe(this.metrics);
```

Comment:
note that by default - without attaching a consumer - this is a memory leak.

Reply:
Do you mean in so far as the bundler keeps producing metrics which get buffered up?

I'm not at all confident in this but... in theory... shouldn't our max buffer size thing take care of that? Assuming `this.metrics` is never consumed then once 100 metrics are buffered it will start throwing away the oldest

Reply:
Nope, that was the memory leak in the frontpage at finn

Reply:
Wasn't that a while back? Or was it from `@podium/metrics`?
This doesn't prevent it?
https://github.com/digitalsadhu/metric-stream/blob/master/lib/client.js#L21

Reply:
Tuesday last week

Reply:
And that clipping seemingly doesn't work... @trygve-lie might have ideas on why?